modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/build.mustache

```diff
@@ -0,0 +1,139 @@
+{{#useAndroidMavenGradlePlugin}}
+group = '{{groupId}}'
+project.version = '{{artifactVersion}}'
+{{/useAndroidMavenGradlePlugin}}
+
+buildscript {
+
+    ext.kotlin_version = '1.5.10'
+
+    ext.swagger_annotations_version = "1.6.2"
+
+    ext.gson_version = "2.8.6"
+
+    ext.volley_version = "1.2.0"
+
+    ext.junit_version = "4.13.2"
+
+    ext.robolectric_version = "4.5.1"
+
+    ext.concurrent_unit_version = "0.4.6"
+
+    repositories {
+        mavenLocal()
+        google()
+        maven {
+            url 'https://dl.google.com/dl/android/maven2'
+        }
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10"
+        classpath 'com.android.tools.build:gradle:4.0.2'
+        {{#useAndroidMavenGradlePlugin}}
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
+        {{/useAndroidMavenGradlePlugin}}
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+{{#useAndroidMavenGradlePlugin}}
+apply plugin: 'com.github.dcendents.android-maven'
+{{/useAndroidMavenGradlePlugin}}
+
+android {
+    compileSdkVersion 30
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
+    }
+    compileOptions {
+        coreLibraryDesugaringEnabled true
+        {{#supportJava6}}
+        sourceCompatibility JavaVersion.VERSION_1_6
+        targetCompatibility JavaVersion.VERSION_1_6
+        {{/supportJava6}}
+        {{^supportJava6}}
+        {{#java8}}
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+        {{/java8}}
+        {{^java8}}
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+        {{/java8}}
+        {{/supportJava6}}
+    }
+    lintOptions {
+        abortOnError false
+    }
+
+    // Rename the aar correctly
+    libraryVariants.all { variant ->
+        variant.outputs.all { output ->
+            if (outputFile != null && outputFileName.endsWith('.aar')) {
+                outputFileName = "${archivesBaseName}-${version}.aar"
+            }
+        }
+    }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
+    }
+}
+
+ext {
+    kotlin_version = '1.5.10'
+    swagger_annotations_version = "1.6.2"
+    gson_version = "2.8.6"
+    volley_version = "1.2.0"
+    junit_version = "4.13.2"
+    robolectric_version = "4.5.1"
+    concurrent_unit_version = "0.4.6"
```

Comment:
@alisters I think all of these versions are no longer needed, since they are also in the top of this file

Reply:
Fixed now